dev-docs/source/Testing/Indirect/DiffractionTests.rst

```diff
@@ -0,0 +1,53 @@
+Indirect Diffraction Testing
+============================
+
+.. contents::
+   :local:
+
+Diffraction
+-----------
+
+*Preparation*
+
+-  You need access to the ISIS data archive
+- ``osiris_041_RES10.cal`` file from the unit test data
+
+**Time required 2-5 minutes**
+
+--------------
+
+#. Open ``Interfaces`` > ``Indirect`` > ``Diffraction``
+#. Make sure ``Instrument`` is set to ``IRIS``
+#. The default Detector Grouping should be ``All``
+#. In ``Run Numbers`` enter 26176
+#. Click ``Run``
+#. The output workspace should have 1 spectra
+#. Change the Detector Grouping too ``Groups`` and enter 4 groups
+#. Click ``Run``
+#. The output workspace should have 4 spectra
+#. Change the Detector Grouping too ``Custom``, enter ``105-108,109-112``
+#. Click ``Run``
+#. The output workspace should have 2 spectra
+#. Still using ``Custom``, try spectra numbers outside of the range 105-112. This should not be allowed.
+#. Using ``Groups``, try more than 8 groups. This should not be allowed.
+#. Using ``File``, try an empty string. This should not be allowed.
+
+**Time required 2-5 minutes**
+
+--------------
+
+#. Open ``Interfaces`` > ``Indirect`` > ``Diffraction``
+#. Make sure ``Instrument`` is set to ``OSIRIS``
+#. Set ``Analyser`` to ``Diffonly``
+#. In ``Run Numbers`` enter 89813
+#. In ``Vanadium File`` enter 89757
+#. In ``Cal File`` load the ``osiris_041_RES10.cal`` file from the unit test data
+#. Select Detector Grouping to be ``All``
+#. Click ``Run``
+#. The output workspace should have 1 spectra
+#. Change the Detector Grouping too ``Groups`` and enter 5 groups
+#. Click ``Run``
+#. The output workspace should have 5 spectra
+#. Change the Detector Grouping too ``Custom``, enter ``3-500,501-962``
+#. Click ``Run``
+#. The output workspace should have 2 spectra
```

Comment:
This makes reference to one output workspace, but I get three ending in `_dRange`, `_q`, `_tof`. The second two changed to five spectra, then they all changed to two.

Reply:
Thanks, I've made those changes by ammending the previous commit.

When I test locally, all three of the output workspaces have the same number of spectra for each step (e.g. 1, then 5, then 2). Are you seeing `_dRange` doesn't have 5 spectra when using ``Groups`` option?

Reply:
I thought not but I was working quickly so very possible it just hadn't updated yet or I misread.

Reply:
Ah maybe, I can't seem to replicate it 🤷‍♂️